### ISSUE TITLE
Select has aria-required on the wrong element

### ISSUE THREAD

**Issue by khalibloo** [ant-design/ant-design#46030]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/hdm23z)

### Steps to reproduce

1. Put a Select inside a FormItem
2. Make FormItem required

### What is expected?

aria-required should not be added to a div tag at the root of Select. It should be added to the input among its children.

### What is actually happening?

aria-required is being added to a div tag. It is likely that other aria-* attributes passed to Select will also just be added to the root element in the Select, instead of the input element.

| Environment | Info |
| --- | --- |
| antd | 5.11.3 |
| React | 18.2.0 |
| System | Windows 10 |
| Browser | Firefox 119 |

**Comment by tarun-sharma-dev-sol** [comment]
No, this error is not resolved yet
